src/styles.css

```
@@ -554,6 +554,11 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
   transform: translateY(0) scale(1);
 }
 
+
+body:has(#prevSectionBtn) #back-to-top-btn {
```

Stylelint (17.12.0)

[error] Expected id selector "`#prevSectionBtn`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)